paper/paper.md

```diff
@@ -121,15 +121,15 @@ some of the taxonomic profilers also come with scripts to convert their output
 into another format and/or merge multiple profiles into a single table, such as
 the Krakentools companion package [@lu_metagenome_2022], these are often focused
 on the specific tool or family of tools. Thus, users would have to become
-proficient in yet another piece of software per tool or family of tools that
-they want to use.
+proficient in yet another piece of software per tool or family of tools just for 
+consistent output files.
 
 [^8]: <https://github.com/bioboxes/rfc/tree/master/data-format>
 [^9]: <https://en.wikipedia.org/wiki/Unix_philosophy#Origin>
 [^11]: <https://library.qiime2.org/plugins/>
 
-Reading a wide range of formats of primarily shotgun-metagenomic profiling tools
-and formats is supported by TAXPASTA, and it is designed to be used as a
+TAXPASTA supports reading a wide range of formats of primarily shotgun-metagenomic
```

Comment:
I agree that active voice is better, but I wanted to avoid starting every paragraph with TAXPASTA.

Reply:
When I read through it I don't think it comes off too repetitive, it's a very short document so it's expected. It's more of a problem when it's every sentence